### PR TITLE
Some NS cleanup

### PR DESCRIPTION
**Pull request by lindsayad**
- Merge `INSFVMethods` into `NavierStokesMethods`
- Wrap methods in `NS` namespace
- Add an `isZero` method in the `MooseUtils` namespace for use in checking whether the velocity is close to 0
- Use this in `INSADTauMaterial` because I'm neurotic and I realized that if the velocity was (-1e-50, 0, 0) then my "fix" would have a zero norm

Refs #22413

### CODE REVIEW

File: framework/include/utils/MooseUtils.h [hunk 1]
```diff
@@ -543,6 +544,36 @@ relativeFuzzyLessThan(const T & var1,
       tol * (std::abs(MetaPhysicL::raw_value(var1)) + std::abs(MetaPhysicL::raw_value(var2)))));
 }
 
+/**
+ * @param value The quantity to test for zero-ness
+ * @param tolerance The tolerance for testing zero-ness. The default is 1e-18
```
Comment (GiudGiud) [inline framework/include/utils/MooseUtils.h:549]: isnt it -24? or is TOLERANCE not the same as libmesh::TOLERANCE?
I see REL_TOLERANCE being defined to 1e-6 not TOLERANCE

File: framework/include/utils/MooseUtils.h [hunk 2]
```diff
@@ -543,6 +544,36 @@ relativeFuzzyLessThan(const T & var1,
...
+ * @return whether the value is (close enough to) zero
```
Comment (GiudGiud) [inline framework/include/utils/MooseUtils.h:550]:
```suggestion
 * @return whether the L_infty norm of the value is (close enough to) zero
```

**Review by GiudGiud — CHANGES_REQUESTED** [review]

File: framework/include/utils/MooseUtils.h (hunk 1 above)
Comment (lindsayad) [reply framework/include/utils/MooseUtils.h:549]: I should clarify the comment that it is 1e-18 for double precision
```c++
#ifdef LIBMESH_DEFAULT_SINGLE_PRECISION
static constexpr Real TOLERANCE = 2.5e-3;
# if defined (LIBMESH_DEFAULT_TRIPLE_PRECISION) || \
     defined (LIBMESH_DEFAULT_QUADRUPLE_PRECISION)
#  error Cannot define multiple precision levels
# endif
#endif

#ifdef LIBMESH_DEFAULT_TRIPLE_PRECISION
static constexpr Real TOLERANCE = 1.e-8;
# if defined (LIBMESH_DEFAULT_QUADRUPLE_PRECISION)
#  error Cannot define multiple precision levels
# endif
#endif

#ifdef LIBMESH_DEFAULT_QUADRUPLE_PRECISION
static constexpr Real TOLERANCE = 1.e-11;
#endif

#if !defined (LIBMESH_DEFAULT_SINGLE_PRECISION) && \
    !defined (LIBMESH_DEFAULT_TRIPLE_PRECISION) && \
    !defined (LIBMESH_DEFAULT_QUADRUPLE_PRECISION)
static constexpr Real TOLERANCE = 1.e-6;
#endif
```

File: framework/include/utils/MooseUtils.h [hunk 3]
```diff
@@ -543,6 +544,36 @@ relativeFuzzyLessThan(const T & var1,
...
+ */
+template <typename T>
+bool
+isZero(const T & value, const Real tolerance = TOLERANCE * TOLERANCE * TOLERANCE)
```
[the remark this answers is not in the document]
Comment (lindsayad) [reply framework/include/utils/MooseUtils.h:574]: I've added handling of containers in most recent commit

File: framework/include/utils/MooseUtils.h [hunk 4]
```diff
@@ -543,6 +544,64 @@ relativeFuzzyLessThan(const T & var1,
       tol * (std::abs(MetaPhysicL::raw_value(var1)) + std::abs(MetaPhysicL::raw_value(var2)))));
 }
 
+// taken from https://stackoverflow.com/a/257382
+template <typename T>
```
Comment (GiudGiud) [inline framework/include/utils/MooseUtils.h:548]:
```suggestion
// Evaluating constexpr (Has_size<T>::value) in a templated method over class T will
// return whether T is a standard container or a singleton
template <typename T>
```

Comment (lindsayad) [inline framework/include/utils/MooseUtils.h:548]:
```suggestion
/**
  * Taken from https://stackoverflow.com/a/257382
  * Evaluating constexpr (Has_size<T>::value) in a templated method over class T will
  * return whether T is a standard container or a singleton
  */
template <typename T>
```

Comment (lindsayad) [reply framework/include/utils/MooseUtils.h:552]: Incorporated suggestion into most recent commit

**Review by GiudGiud — APPROVED** [review]

File: framework/include/utils/MooseUtils.h (hunk 1 above)
Comment (GiudGiud) [reply framework/include/utils/MooseUtils.h]: sounds good.
I must have pulled the wrong one when I checked

**Pull request merged**